application/blueprints/main/views.py

```diff
@@ -28,6 +28,13 @@ def advisory_group():
     return redirect(url_for("project.planning_applications"), code=301)
 
 
+@main.route("/get-involved")
+def get_involved_in_designing_data():
+    path = "pages/get-involved-in-designing-data.md"
+    page = "get-involved-in-designing-data"
+    return render_template("pages/scaffold.html", path=path, page=page)
```

Comment:
We don't need this as page route covers this

Reply:
```suggestion
```

Reply:
if you want a short url - it's better to just rename the file to `get-involved` or use `get-involved-in-designing-data`.

or use the new route entry to add the following (line 205):

```
redirects = {
    "how-to-contribute-to-the-data-design-process": "get-involved-in-designing-data",
    "get-involved": "get-involved-in-designing-data",
}
```

Reply:
cc @fabiafowler